File: app/src/main/java/mohsin/reza/propertyapp/binding/FragmentDataBindingComponent.java
package mohsin.reza.propertyapp.binding;

import android.databinding.DataBindingComponent;
import android.support.v4.app.Fragment;


public class FragmentDataBindingComponent implements DataBindingComponent {
    private final FragmentBindingAdapters adapters;

    public FragmentDataBindingComponent(Fragment fragment) {
        this.adapters = new FragmentBindingAdapters(fragment);
    }

    @Override
    public FragmentBindingAdapters getFragmentBindingAdapters() {
        return adapters;
    }
}

File: app/src/main/java/mohsin/reza/propertyapp/binding/FragmentDataBindingComponent.kt
package mohsin.reza.propertyapp.binding

import android.support.v4.app.Fragment
import android.databinding.DataBindingComponent

/**
 * Created by inteliment on 4/05/2018.
 */
class FragmentDataBindingComponent(fragment: Fragment) : DataBindingComponent {
    private val adapter = FragmentBindingAdapters(fragment)

    override fun getFragmentBindingAdapters() = adapter
}